content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -522,24 +522,33 @@ This means that support is still available but will be removed in a future Kuber
 * [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume
   (**deprecated** in v1.28)
 
-Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
+The following PersistentVolume types are supported through the `CSIMigration` feature gate and require the corresponding CSI driver to be installed in the cluster. Kubernetes offers built-in APIs to facilitate their use.
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
```

Comment:
There are these volume plugins / corresponding CSI drivers missing:

* AzureFile CSI migration went GA in 1.26
* vSphere CSI migration went GA in 1.26

IMO it would make more sense to have a table with in-tree volume plugin name, link to the CSI driver, and  Kubernetes version, with generic `this volume type is **only** available via a separate CSI driver 
  as Kubernetes' core no longer provides built-in support.` below it.

Reply:
> IMO it would make more sense to have a table with in-tree volume plugin name, link to the CSI driver, and Kubernetes version, with generic `this volume type is **only** available via a separate CSI driver as Kubernetes' core no longer provides built-in support.` below it.

PRs very welcome (from any contributor)

Reply:
@sftim @jsafrane Unfortunately, I do not have the capacity currently to make this formatting change.
I request to not include this formatting change in the scope of this PR. If needed it can be opened as an enhancement issue and a contributor who would have the capacity could pick it up.

Reply:
As commented early by @jsafrane , AzureFile and vSphere are missing.